radio/src/myeeprom.h

```diff
@@ -422,7 +422,8 @@ PACK(typedef struct {
   int8_t  rfProtocol:4;
```

Comment:
Oh yes I remember why I put a int there, thanks! I should add comments I know ...